CHANGES.rst

```diff
@@ -461,9 +461,9 @@ Bug fixes:
 
 
 - Added url field to Actions (#817)
-- Update statictime tests following changes to p.a.disucssion (see 
-  https://github.com/plone/plone.app.discussion/pull/204) - [instification] (#1520)
-- Update @portrait endpoint to use sanitized user id [instification] (#1524)
+- Updated ``statictime`` tests following changes to ``p.a.discussion`` (see
+  https://github.com/plone/plone.app.discussion/pull/204) - @instification (#1520)
```

Comment:
One more and it's good!

```suggestion
  https://github.com/plone/plone.app.discussion/pull/204). @instification (#1520)
```

Reply:
@stevepiercy this is something I am always wondering about. In German, you do not use punctuation when using bullet points. Is the way to go here to always use "." at the end of each changelog entry? I am ok with it either way, I just want to have one recommended way. :)

Reply:
Either is acceptable in English, depending on the context. Usually the change notes are in fact complete sentences, which should end in a period, regardless of whether it is in a list. When the bulleted list is an actual list of items and not a sentence, such as:

- Item1
- Item2
- Item3

...then you would not use a period at the end of the item.

See the Microsoft Style Guide for lists: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists

Reply:
@stevepiercy thanks for the clarification Steve! I will use a period from now on in changelog entries.